### ISSUE TITLE
Fedora 37 - error when trying to open heic file

### ISSUE THREAD

**Issue by supertoni3c** [strukturag/libheif#765]
Since a few weeks, I'm not able to open heif/heic files at Fedora 37. I think it was caused by an update pushed around Christmas. I saw some reports about this issue but as far as I can see nobody report this bug here. 

I see this information when I try to open the heic file via Gnome image viewer.
![Screenshot from 2023-01-15 17-09-43](https://user-images.githubusercontent.com/105498130/212552396-e1977491-063d-4d5a-8664-9559de72b70d.png)

**Comment by farindk** [comment]
Usually, when people reported this, it was in fact a JPEG file with a wrong file extension.
If you believe that this is not the case, you can attach an example image so that I can have a look.

**Comment by supertoni3c** [comment]
I can't attach the example because heic is not supported here. I can only share it from my cloud storage:

https://kdrive.infomaniak.com/app/share/540285/4ea9ffaf-6f44-40f0-98b7-5fb8f5782a6f

**Comment by farindk** [comment]
Link doesn't exist...

**Comment by supertoni3c** [comment]
It expired, this one should be active till next week:

https://kdrive.infomaniak.com/app/share/540285/7f33e186-05ed-40af-b578-8859753c7299

**Comment by farindk** [comment]
Works fine with libheif:
````
$ heif-convert 20230118_171147_5270.heic out.jpg
File contains 1 image
Written to out.jpg
````

As there were no changes in the gdk-pixbuf interface, I assume it's an installation issue.

**Comment by supertoni3c** [comment]
Since about a week it works again.

**Comment by leigh123linux** [comment]
> Since about a week it works again.

It isn't fixed as such, I switched the build back to using autotools.
Trying to use cmake for the build is a no-go due to missing linking.

https://bugzilla.rpmfusion.org/show_bug.cgi?id=6532

**Comment by kmilos** [comment]
Out of curiosity - since there is the plugin infrastructure available now, would it be possible to get libheif into Fedora proper w/ AV1 plugins, and have separate HEVC plugins in RPMFusion?

**Comment by leigh123linux** [comment]
> Out of curiosity - since there is the plugin infrastructure available now, would it be possible to get libheif into Fedora proper w/ AV1 plugins, and have separate HEVC plugins in RPMFusion?


I won't be submitting libheif  for fedora repos, others have shown some interest in doing it.
If it's moved to fedora the new maintainer will also need to maintain the rpmfusion counterpart as I dislike split packages. 
Currently the plugin system is totally broken due to not linking with libheif.so.

**Comment by farindk** [comment]
@leigh123linux The linking issue is fixed in v1.15.0
